aladdin.sh

```diff
@@ -125,7 +125,14 @@ function enter_minikube_env() {
 function _extract_cluster_config_value() {
     local value
     value="$1"
-    jq -r ".$value" "$ALADDIN_CONFIG_DIR/$CLUSTER_CODE/config.json"
+    output=$(jq -r ".$value" "$ALADDIN_CONFIG_DIR/$CLUSTER_CODE/config.json")
+    if [[ "$output" == null ]]; then
+        output=$(jq -r ".$value" "$ALADDIN_CONFIG_DIR/default/config.json")
+    fi
+    if [[ "$output" == null ]]; then
+        output=$(jq -r ".$value" "$ALADDIN_CONFIG_DIR/config.json")
+    fi
+    echo "$output"
```

Comment:
Change this function to check configurations in cluster, default, and overall config files